### PR TITLE
Various fixes for the model panel

### PR DESCRIPTION
**Pull request by RosaRomeroGomez**
**What**
- Made counters for the variables panel consistent with the counters in the parameters panel
- Fixed issues with the highlighting of nodes in the Model panel. Double clicking adds/removes variables and parameters from the panels. 
- Commented out the tooltips until we fix them.

### CODE REVIEW

File: client/src/views/Simulation/components/VariablesPanel.vue [hunk 1]
```diff
@@ -41,7 +41,7 @@
               type="button"
               @click="setSimVariableVisibility(plot.metadata.name)"
             >
-              <font-awesome-icon :icon="['fas', (plot.hidden ? 'eye' : 'eye-slash')]" />
+              <font-awesome-icon :icon="['fas', (plot.hideen ? 'eye' : 'eye-slash')]" />
```
Comment (YohannParis) [inline client/src/views/Simulation/components/VariablesPanel.vue:44]: typo here

File: client/src/graphs/svg/renderers/EpiRenderer.ts [hunk 2]
```diff
@@ -196,15 +196,12 @@ export default class EpiRenderer extends SVGRenderer {
...
     const chart = (this as any).chart;
-    const nodes = subgraph.nodes;
-
     chart.selectAll('.node-ui').each(function (d) {
-      const isHighlighted = nodes.map(node => node.id).includes(d.id);
+      const isHighlighted = nodesList.map(node => node.id).includes(d.id);
       d3.select(this).select('rect')
-        .style('stroke', isHighlighted ? Colors.HIGHLIGHT : Colors.STROKE)
-        .style('stroke-width', DEFAULT_STYLE.node.strokeWidth + (isHighlighted ? 3 : 0));
+        .style('fill', isHighlighted ? Colors.NODES.EDITED : EpiRenderer.calcNodeColor(d));
```
Comment (mj3cheun) [inline client/src/graphs/svg/renderers/EpiRenderer.ts:204]: Did you mean to revert back to node fill highlighting? Now the simulate page looks like this.

Screen Shot 2021-07-20 at 9 20 15 AM

File: client/src/types/types.ts [hunk 3]
```diff
@@ -53,6 +53,7 @@ interface SimulationParameter extends Donu.ModelParameter {
 type SimulationVariableValues = Array<{x: number, y: number}>;
 
 interface SimulationVariable extends Donu.ModelVariable {
+  edited?: boolean,
```
Comment (YohannParis) [inline client/src/types/types.ts:56]: Why do we need an edited on the Variables panel?

**Review by YohannParis — APPROVED** [review]

**Comment by mj3cheun** [comment]
Looks like if you use the hide/show button and the model representation to select/deselect nodes, its possible to make the model representation go out of sync with the parameters/variables panel

**Comment by YohannParis** [comment]
I get no charts on the variables panel.
You can see on the top right I have 1 run, and I can see that Galois sends the results.

Screen Shot 2021-07-20 at 09 26 56

File: client/src/graphs/svg/renderers/EpiRenderer.ts (hunk 2 above)
Comment (RosaRomeroGomez) [reply client/src/graphs/svg/renderers/EpiRenderer.ts:204]: image
This is how it looks for me @mj3cheun

File: client/src/types/types.ts (hunk 3 above)
Comment (RosaRomeroGomez) [reply client/src/types/types.ts:56]: We should also be able to add and remove variables from the model visualization to the Variables panel. Does this make sense @YohannParis ?

File: client/src/views/Simulation/components/VariablesPanel.vue (hunk 1 above)
Comment (RosaRomeroGomez) [reply client/src/views/Simulation/components/VariablesPanel.vue:44]: Fixed

File: client/src/graphs/svg/renderers/EpiRenderer.ts (hunk 2 above)
Comment (RosaRomeroGomez) [reply client/src/graphs/svg/renderers/EpiRenderer.ts:204]: So yes, this was intentional as discussed over slack.

File: client/src/types/types.ts (hunk 3 above)
Comment (YohannParis) [reply client/src/types/types.ts:56]: Rosa explained that now Variables behaves like Parameters and you need to double-click on the model node to display them on the Variables panel.

**Comment by YohannParis** [comment]
- Added the Add/remove all on the variables panel.
- Add actions and mutations on the simulation store to handle those request.
- Hide the `Show/Hide` buttons on the parameters panel
- Update the variable button to _remove_ instead of _hide_

https://user-images.githubusercontent.com/636801/126348519-5ffb0883-7456-4253-952a-fe0c6ba54e99.mov

**Review by YohannParis — APPROVED** [review]

**Pull request merged**